ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/EthSignerBaseCommand.java

```diff
@@ -107,6 +107,25 @@ public File getStorePasswordFile() {
     }
   }
 
+  static class TlsClientAuthentication {
+    @Option(
+        names = "--tls-known-clients-file",
+        description = "Path to a file containing the fingerprints of authorized clients.",
+        arity = "1",
+        required = true)
+    private final File tlsKnownClientsFile = null;
+
+    @SuppressWarnings("UnusedVariable")
+    @Option(
+        names = "--tls-disable-client-authentication",
```

Comment:
I don't think we use 'disable' for any other options https://github.com/hyperledger/besu/blob/master/CLI-STYLE-GUIDE.md#flags

Reply:
does this make it inconsistent?

Reply:
I agree - gut I couldn't think of how to do this as an "enable" - given enable is the default.
Also - I didn't want to be able to write --tls-disable-client-authentication=false; and thus the 'arity' is 0 - i.e. this acts as a flag option...
SPOSE we could change the language and make it "--tls-allow-any-client"?